vllm/v1/cudagraph_dispatcher.py

```diff
@@ -113,21 +162,19 @@ def dispatch(
         if not self.keys_initialized:
             return CUDAGraphMode.NONE, None
 
-        non_uniform_key = batch_descriptor.non_uniform
-        # if a batch use cascade attention, bypass checking full cudagraphs
-        if not use_cascade_attn:
-            # check if key exists for full cudagraph
-            if batch_descriptor in self.cudagraph_keys[CUDAGraphMode.FULL]:
-                return CUDAGraphMode.FULL, batch_descriptor
+        num_tokens, uniform_decode = (
+            batch_descriptor.num_tokens,
+            batch_descriptor.uniform_decode,
+        )
```

Comment:
Can we make this a new data structure? Or can we just use `BatchDescriptor` with `num_reqs=None`?